src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -93,6 +93,10 @@ StatusModal {
                     font.pixelSize: 28
                     font.family: Theme.palette.baseFont.name
                     color: Theme.palette.directColor1
+                    Keys.onPressed: {
+                                        if (event.key === Qt.Key_Enter || event.key === Qt.Key_Return)
+                                          event.accepted = true
+                                        }
```

Comment:
Could you pls use `CMD+I` for it?

Reply:
sure thing, i mistakenly forgot to do it

Reply:
@borismelnik this is done, sorry about it